domain/domain.go

```diff
@@ -392,8 +392,10 @@ func (m *MockFailure) getValue() bool {
 	return m.val
 }
 
-type etcdBackend interface {
+// EtcdBackend is used for judging a storage is a real TiKV.
```

Comment:
Well, `kv.Storage` is an common interface, we'd better keep it clean.
But some additional work is necessary for specific storage engine, so it is defined.
We use it like this:

```
if rawStore, ok := store.(EtcdBackend); ok {
    // do something...
}
```